.clang-format

```diff
@@ -61,7 +61,7 @@ PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 100
 PointerAlignment: Right
 ReflowComments: true
-SortIncludes: true
+SortIncludes: false # sysinc.h must be before module.h
```

Comment:
Just a note. Sections of code can be [protected](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code) from auto formatting like so:
```
/* clang-format off */
#include "sysinc.h"
#include "module.h"
/* clang-format on */
```

Reply:
thanks, changed